### PR TITLE
Add specs for rb_struct_define_under

### PR DESCRIPTION
**Pull request by kaorimatz**

### CODE REVIEW

File: optional/capi/struct_spec.rb [hunk 1]
```diff
@@ -57,6 +57,37 @@
...
+      # Verify that attributes are on an instance basis
+      CApiStructSpecs::CAPIStruct.new.b.should be_nil
+    end
+
+    it "creates a constant scoped under the namespace of the given class" do
+      CApiStructSpecs.const_defined?(:CAPIStruct).should be_true
+    end
+
+    it "returns the member names as Symbols" do
+      @struct.members.sort.should == [:a, :b, :c]
```
Comment (eregon) [inline optional/capi/struct_spec.rb]: Are they not in order otherwise? I think `Struct#members` guarantees so the `sort` should not be needed.

Comment (kaorimatz) [inline optional/capi/struct_spec.rb]: Nice catch, fixed in 6e0cb28.

File: optional/capi/struct_spec.rb [hunk 2]
```diff
@@ -57,6 +57,37 @@
...
+      instance.c.should == 3
+    end
+
+    it "has a value of nil for the member of a newly created instance" do
+      # Verify that attributes are on an instance basis
+      CApiStructSpecs::CAPIStruct.new.b.should be_nil
+    end
+
+    it "creates a constant scoped under the namespace of the given class" do
+      CApiStructSpecs.const_defined?(:CAPIStruct).should be_true
```
Comment (kaorimatz) [inline optional/capi/struct_spec.rb]: Wow I didn't know that, thank you! fixed in 077cc34.

**Comment by kaorimatz** [comment]
@eregon Thank you for reviewing!

**Comment by eregon** [comment]
@kaorimatz Thank you for new specs!

**Pull request merged**